src/quote/Quote.ts

```diff
@@ -108,3 +108,81 @@ export function createAgreedQuote(
     claimerSignature: signature,
   }
 }
+
+export function compressCost(quote: IQuote): any {
```

Comment:
Change request: 
You're passing the whole `quote` object to this `compressCost` function but it doesn't need it; you can pass it simply a `cost`. It will also make the API clearer!

Reply:
Updated. 

Do you prefer `ICostBreakdown` or `IQuote['cost']`?

Reply:
`ICostBreakdown` as you have it is good 👍